fairlearn/postprocessing/_thresholder.py

```diff
@@ -0,0 +1,241 @@
+# Copyright (c) Fairlearn contributors.
+# Licensed under the MIT License.
+
+import numpy as np
+from sklearn import clone
+from sklearn.base import BaseEstimator, MetaEstimatorMixin
+from sklearn.exceptions import NotFittedError
+from sklearn.utils.validation import check_is_fitted
+from warnings import warn
+
+
+from postprocessing._threshold_operation import ThresholdOperation

```

Comment:
```suggestion
from ._threshold_operation import ThresholdOperation

```
at least locally that's what makes this all work for me